appinventor/buildserver/src/com/google/appinventor/buildserver/BuildServer.java

```diff
@@ -155,6 +156,8 @@ public void report(int progress) {
 
   private static final CommandLineOptions commandLineOptions = new CommandLineOptions();
 
+  private final String AAB_TRUE_VALUE = "1";
```

Comment:
I decided to specify here the value that must be received in the request to build the AAB, thinking about the future if it is wanted to change it to "true" or something ekse.

Reply:
Another option would be to ignore the value, and build the AAB if the `aab` param is received

Reply:
Also, now that Play Store will enforce to upload AABs, maybe the normal PlayApp command could build both AAB and APK (from bundletool)